engine/src/main/java/org/terasology/persistence/WorldDumper.java

```diff
@@ -52,4 +54,23 @@ public void save(Path file) throws IOException {
             EntityDataJSONFormat.write(world, writer);
         }
     }
+
+    /***
+     * Save World entities, which only contain some of Components
+     * @param file
+     * @param filterComponents
+     * @throws IOException
```

Comment:
Please add param and exception descriptions here, they also are very helpful!

Reply:
Done

Reply:
👍